### DISCUSSION TITLE
10 different threads wanting ability to mute or block people from the feed

### DISCUSSION THREAD

**Discussion by h-unterp** [community/community#20310] [Other Feature Feedback, Questions, & Ideas] [answered]
https://github.com/orgs/github-community/discussions/13282
https://github.com/orgs/github-community/discussions/13144
https://github.com/orgs/github-community/discussions/16860
https://github.com/orgs/github-community/discussions/14291
https://github.com/orgs/github-community/discussions/13183
https://github.com/orgs/github-community/discussions/16175
https://github.com/orgs/github-community/discussions/13390
https://github.com/orgs/github-community/discussions/15698
https://github.com/orgs/github-community/discussions/13161
https://github.com/orgs/github-community/discussions/13179

![Screen Shot 2022-07-10 at 11 54 56 PM](https://user-images.githubusercontent.com/772917/178185571-435fa774-285f-443a-8074-c4db5da3ae19.png)

Literally 😉

**Comment by ShadowMarker789** [comment]
>I have done this thing 

>Has not done the thing 

That sounds exactly like a lot of the IT Support, Ops Support. There have been too many times I've asked "Have you done X?" with response "Yes, I have done X" when they had in fact not done X.

**Comment by drewboswell** [comment, marked as answer]
Seems like this one is the most active at the moment: https://github.com/orgs/community/discussions/13144
